test/AaveAtokenFork.t.sol

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: Unlicensed
+pragma solidity >=0.8.10;
+
+import { Test } from "forge-std/Test.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { ERC20 } from "openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
+
+import { IAToken } from "aave-v3-core/contracts/interfaces/IAToken.sol";
+import { AToken } from "aave-v3-core/contracts/protocol/tokenization/AToken.sol";
+import { IPool } from 'aave-v3-core/contracts/interfaces/IPool.sol';
+import { ConfiguratorInputTypes } from 'aave-v3-core/contracts/protocol/libraries/types/ConfiguratorInputTypes.sol';
+import { PoolConfigurator } from 'aave-v3-core/contracts/protocol/pool/PoolConfigurator.sol';
+import { DataTypes } from 'aave-v3-core/contracts/protocol/libraries/types/DataTypes.sol';
+import { AaveOracle } from 'aave-v3-core/contracts/misc/AaveOracle.sol';
+
+import { GovToken } from "./GovToken.sol";
+
+import { Pool } from 'aave-v3-core/contracts/protocol/pool/Pool.sol';
+import "forge-std/console2.sol";
+
+contract AaveAtokenForkTest is Test {
+  uint256 forkId;
+
+  IAToken aToken;
+  GovToken govToken;
+  IPool pool;
+
+  address dai = 0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1;
+  address weth = 0x4200000000000000000000000000000000000006;
+
+  function setUp() public {
+    // We need to use optimism for Aave V3 because it's not (yet?) on mainnet.
+    // https://docs.aave.com/developers/deployed-contracts/v3-mainnet
+    uint256 optimismForkBlock = 26332308; // The optimism block number at the time this test was written.
+    forkId = vm.createSelectFork(vm.envString("OPTIMISM_RPC_URL"), optimismForkBlock);
```

Comment:
I'd advocate for using `[rpc_endpoints]` and `vm.rpcUrl` instead of the env var approach as our default approach for forking tests, as detailed here: https://twitter.com/msolomon44/status/1564742781129502722

Reply:
I'm fine with doing this, but do you mind explaining why it's better? Seems to be functionally equivalent and `rpcUrl("optimism")` feels no simpler nor more self-documenting than `envString("OPTIMISM_RPC_URL")`.

On the other hand, I guess if you fat-finger the rpc name you do get a helpful message.

Reply:
Done in 8a07568

Reply:
You are correct they are functionally equivalent, it's mainly a convention thing. The other benefit is that you can look at `foundry.toml` to learn how to setup that project, e.g what RPC URLs are used/does it need. In the future looking at the config file can also give insight into which files/tests are most core/critical, e.g. which are more heavily fuzzed (https://github.com/foundry-rs/foundry/issues/3062)